measurements/api/measurements.py

```diff
@@ -147,8 +151,56 @@ def list_files(
         'results': results
     })
 
+
+
+def get_one_fastpath_measurement(measurement_id, download):
+    """Get one measurement from the fastpath table by measurement_id,
+    fetching the file from the fastpath host
+    """
+    log = current_app.logger
+    tid = measurement_id[len(FASTPATH_MSM_ID_PREFIX) :]
+
+    path = "/measurements/{}.json.lz4".format(tid)
+    log.info(
+        "Incoming fastpath query %r. Fetching %s:%d%s",
+        measurement_id,
+        FASTPATH_SERVER,
+        FASTPATH_PORT,
+        path,
+    )
+    conn = http.client.HTTPConnection(FASTPATH_SERVER, FASTPATH_PORT)
+    log.debug("Fetching %s:%d %r", FASTPATH_SERVER, FASTPATH_PORT, path)
+    conn.request("GET", path)
+    r = conn.getresponse()
```

Comment:
Is there a reason to use `http.client` instead of the `requests` library which is already a dependency (see: https://github.com/ooni/api/blob/cbf7a3925812d01af798e70cb3f71de153dffe2f/measurements/api/measurements.py#L184)?

Reply:
https://docs.python.org/3.7/library/urllib.request.html and http.client provide most of the same features without needing an external dependency. We could slowly move to urllib.request over time and drop the dep.

Reply:
Ok seems reasonable. Let's keep this as-is then for the time being and maybe consider dropping requests in the future.

See: https://github.com/ooni/api/issues/130